src/components/AddResourceModal/AddResourceModalV2.jsx

```diff
@@ -92,25 +95,70 @@ export default function AddResourceModalV2(props) {
     });
   };
 
-  const textFieldChangeHandler = e => {
-    setValues(prevValues => {
-      // the address textbox is set with react-hook-form setValue() when user clicks "use my location instead",
-      //  which doesn't fire an event. When that happens, a string is passed instead of an event object, so this
-      // conditional expression handles that
-
-      if (e.target) {
-        return {
-          ...prevValues,
-          [e.target.name]: e.target.value
-        };
+  const debounce = (func, delay) => {
```

Comment:
Can we move this to a new file `src/utils/debounce.js`?

Reply:
Hi @RNR1 - I moved the debounce function to `src/utils/debounce.js` and updated the import statement accordingly. Also, I removed the console.log statements as requested. Please review the changes. Thanks!